DbgProvider/public/Debugger/DbgEngDebugger.cs

```diff
@@ -2094,6 +2114,106 @@ public DbgModuleInfo GetModuleByAddress( ulong addrInModule )
             } );
         } // end GetModuleByName()
 
+        //Retrieves the ntdll module associated with what DbgEng considers to be the native architecture
```

Comment:
>//Retrieves [](start = 8, length = 11)

Is there an .editorconfig setting for having a space after the opening "`//`"?

Reply:
It seems there is not. Resharper has added an impressive suite of editorconfig settings to their formatting, including angle bracket type parameter settings, so I've added those to the editorconfig, but while they have a `csharp_space_before_trailing_comment` for a space between code and the opening //, there's nothing for the right side.